### ISSUE TITLE
Several Broken Recipes

### ISSUE THREAD

**Issue by MadSciSlack** [CodeF53/Create-Renewable-Ores#4]
Hey, these have been broken for as long as I've used the datapack, which is since it was published.  These all show up as "failed to parse recipe."  The reason they don't work is because the outputs all are named wrong.

Create doesn't use "crushed_'x'_ore".  They use "crushed_raw_'x'".

create-renewables:gold_brimstone  - Claims to not be able to find create:crushed_gold_ore
create-renewables:copper_deepslate  - Claims to not be able to find create:crushed_copper_ore
create-renewables:copper  - Claims to not be able to find create:crushed_copper_ore
create-renewables:zinc_deepslate  - Claims to not be able to find create:crushed_zinc_ore
create-renewables:iron_deepslate  - Claims to not be able to find create:crushed_iron_ore
create-renewables:iron  - Claims to not be able to find create:crushed_iron_ore
create-renewables:ores_zinc  - Claims to not be able to find create:crushed_zinc_ore
create-renewables:gold_nether  - Claims to not be able to find create:crushed_gold_ore
create-renewables:gold  - Claims to not be able to find create:crushed_gold_ore
create-renewables:nylium_and_wart/wailing_wart_block  - Claims that the output is empty, but probably can't find the item.
create-renewables:nylium_and_wart/embur_wart_block  - Claims that the output is empty, but probably can't find the item.
create-renewables:gold_nether_blue  - Claims to not be able to find create:crushed_gold_ore

In other words, you've had this out for over a year and it _really_ seems like you've never checked the item IDs, never checked for errors, and never done an update.  Needs improvement.  Literally and immediately.
